packages/serverless/src/model/abstract/document.ts

```diff
@@ -314,16 +158,10 @@ possible field kinds explicitly to ensure correctness.`
   }
 
   public async findHistoryOne(docId: string, session?: ClientSession) {
-    const documents = this.find({ docId }, session);
-    return documents;
-  }
-
-  public async find(filter?: Filter<any>, session?: ClientSession) {
-    logger.debug(`ModelDocument::find()`, { filter });
-    return this.collection.find(filter || {}, { session }).toArray();
+    return this.find({ docId }, { session });
   }
 
-  public async countActiveDocuments(filter?: Filter<any>) {
+  public async countActiveDocuments(filter?: Filter<TDocumentRecordNullable>) {
```

Comment:
```suggestion
  public async countActiveDocument(filter?: Filter<TDocumentRecordNullable>) {
```

Reply:
I think this case should stay plural because we're counting